docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -612,6 +612,28 @@ quarkus.kubernetes.app-config-map=<name of the config map containing the configu
 When these properties are used, the generated manifests will contain everything required.
 The application config volumes will be created using path: `/mnt/app-secret` and `/mnt/app-config-map` for secrets and configmaps respectively.
 
+[NOTE]
+====
+
+For this to work, the secret or configmap must contain an entry with a configuration file name recognized by Quarkus (e.g. `config.properties`)
```

Comment:
My understanding is that the `config.properties` file name is of no interest? What's of interest is that the content is a valid `.properties` file? Or am I mistaken?

Reply:
I also assumed that, but actually Smallrye Config ignores any files without the `.properties` suffix when it processes a directory.

Reply:
To reproduce:

- put critical configuration properties in `./configfiles/config`
- issue `mvn quarkus:dev -Dsmallrye.config.locations=./configfiles`

The properties will not be picked up.

Then rename `./configfiles/config` to `./configfiles/config.properties` and try again. It should work.